### DISCUSSION TITLE
External hard drive?

### DISCUSSION THREAD

**Discussion by MiniSowgro** [Xpl0itU/WiiUDownloader#57] [General]
Using WiiUDownloader on Steam Deck desktop mode, and it works perfectly, but when choosing a place to download files to my external microSD card will not show. Is there anything i can do to remedy this? Sorry if this sounds dumb, I’m relatively new to Linux.

**Comment by Xpl0itU** [comment]
It’s up to the os’s file picker, not WiiUDownloader
